first.css

```diff
@@ -0,0 +1,112 @@
+body {
+  height: 7000px;
+}
+
+.floating-subscribe-wrapper 
+{
+    background: #fff;
+    background-color: rgb(255, 255, 255);
+    box-shadow: 0 2.75335px 44.0535px rgb(0 0 0 / 12%);
+    border-radius: 5px;
+    padding: 28px;
+}
+.hide {
+  opacity: 0;
+ 
+}
+.show {
+  opacity: 1;
+}
+
+
+.heading
+{
+	text-align: left;
+	 font-size: 20px;
+   font-family: Poppins;
+}
+
+.floating-subscribe-wrapper .floating-subscribe-close
+ {
+    position: absolute;
+    right: 24px;
+    top:34px;
```

Comment:
This should be bottom I guess. Float card bottom la position agirukanum da. When checking for me it was floating at center because of this top property